pallets/vested-rewards/src/lib.rs

```diff
@@ -606,6 +812,143 @@ pub mod pallet {
             )?;
             Ok(().into())
         }
+
+        #[allow(unused_variables)]
+        #[transactional]
```

Comment:
`transactional` is not needed here, all our extrinsics are transactional by default

Reply:
so removed for all extrinsics